lib/linter.sh

```diff
@@ -213,7 +213,7 @@ LINTER_NAMES_ARRAY['ARM']="arm-ttk"
 LINTER_NAMES_ARRAY['BASH']="shellcheck"
 LINTER_NAMES_ARRAY['BASH_EXEC']="bash-exec"
 LINTER_NAMES_ARRAY['CLOJURE']="clj-kondo"
-LINTER_NAMES_ARRAY['CLOUDFORMATION']="cfn-lint"
+LINTER_NAMES_ARRAY['CLOUDFORMATION_CFN_NAG']="cfn-lint"
```

Comment:
The right side of the equals sign should be something about `cfn-nag` (or `cfn_nag`?) but not `cfn-lint`